tests/cloner_test.go

```diff
@@ -574,6 +575,75 @@ var _ = Describe("all clone tests", func() {
 						string(dv.Status.Progress) == "N/A"
 				}, timeout, pollingInterval).Should(BeTrue(), "DV Should succeed with storage.prePopulated==pvcName")
 			})
+
+			DescribeTable("Should clone with empty volume size without using size-detection pod",
+				func(sourceVolumeMode v1.PersistentVolumeMode, targetVolumeMode v1.PersistentVolumeMode) {
```

Comment:
if you have multiple arguments of the same type you can do arg1, arg2, arg3, etc type. so in this case it would be:
```go
func(sourceVolumeMode, targetVolumeMode v1.PersistentVolumeMode)
```